### ISSUE TITLE
"Contextual Help" handling (standardized)

### ISSUE THREAD

**Issue by ninavizz** [QubesOS/qubes-issues#6669]
**The problem you're addressing (if any)**
Often times it serves user education to let folks know how a thing works or what a term-of-art means, in the context of a task within the UI. Nowadays, folks often don't even read user manuals—as many digital experiences are built with such intuitive experiences and enough contextual help nodes built-in, that documentation is rendered not necessary. 

Many issues here have spoken to a desire for a thing to be explained, contextually. To date however, no design standard has been created for contextual help to be consistently implemented in the Qubes UI.

**Describe the solution you'd like**
A blue "i" icon to the left of an element that will be explained. Upon hover, an explainer bubble will appear. 

Part one of the solution is the interaction pattern. Part two, is how the text in the bubble is structured. 

Part one is often hacked with using an element's alt-text field. However, that hack [violates accessibility best practices](https://webaim.org/techniques/alttext/) and should be avoided.

<img width="614" alt='A cursor hovers over a blue "i" icon, and a black bubble with text propogates to the left' src="https://user-images.githubusercontent.com/8262612/120904497-7aa78d00-c601-11eb-88f4-3670a9384fbe.png">

**Where is the value to a user, and who might that user be?**
Helping folks learn Qubes OS as they go along. Also including hyperlinks to Qubes OS docs, to nudge users to learn more about a topic than a <100cc blurb can adequately explain. Moreso, the latter. 

More folks need to read the docs, and more will if they are referenced from contextual help bubbles.

**Describe alternatives you've considered**
RTFD. 
Tears. 

**Related, [non-duplicate](https://www.qubes-os.org/doc/reporting-bugs/#new-issues-should-not-be-duplicates-of-existing-issues) issues**
Yes, many. To start, this specific example is suggested for #6366

**Comment by andrewdavidwong** [comment]
Closely related: #2211
